test/powershell/Modules/Microsoft.PowerShell.Utility/MarkdownCmdlets.Tests.ps1

```diff
@@ -375,6 +375,34 @@ bool function()`n{`n}
             }
 
 
+            $options.Link | Should -BeExactly "[4;38;5;117m"
+            $options.Image | Should -BeExactly "[33m"
+            $options.EmphasisBold | Should -BeExactly "[1m"
+            $options.EmphasisItalics | Should -BeExactly "[36m"
+        }
+
+        It "Verify PSMarkdownOptionInfo is defined in module scope" {
+
+            $mod = Get-Module Microsoft.PowerShell.Utility
+            $options =  & $mod { $PSMarkdownOptionInfo }
```

Comment:
Better add one check for `$PSMarkdownOptionInfo -eq $null` before this line, to make sure that the variable is not leaked into the global scope or current scope.